server/index.js

```diff
@@ -8,8 +8,15 @@ require('dotenv').config();
 app.use(morgan('dev'));
 app.use(express.json());
 
+
 app.use(express.static(path.join(__dirname, '..', 'dist')));
 
+app.get('*', (req, res) => {
```

Comment:
Hey Eddy!
Before I accept this pull request, could you change the * in this app.get to only accept requests from the specific routes? (what we talked about earlier today)